docs/contribute/contribute.adoc

```diff
@@ -4,126 +4,169 @@ This document is meant for new OpenSCAP contributors and it describes how to
 Before you start working on a fix it's a good practice to leave a
 comment in the issue that you work on the fix so other contributors know that
 the fix is in progress.  Next thing you have to do is to fork the OpenSCAP
-repository. You can do that by pressing the **Fork** button in the top-right
+repository. You can do that by pressing the *'Fork'* button in the top-right
 corner on the Github's link:https://github.com/OpenSCAP/openscap[OpenSCAP page].
 It will create a copy of the original repository which you can use for
 proposing changes.
 
+
 == Choose the right branch
-Before you start working of the fix it's necessary to determine which branch the
+Before you start working on the fix it's necessary to determine which branch the
 fix will go into. If you are not familiar with the OpenSCAP's branches or
 versions yet please have a look at the link:versioning.adoc[versioning]
 document. Be aware that the fact that an issue description says that the fix
-should go to a _maint-1.2_ branch doesn't have to be true. It's a good practice
+should go to the `maint-1.2` branch doesn't have to be true. It's a good practice
 to investigate the correct branch first or ask experienced developers on our
 FreeNode IRC channel called `#openscap` or
 link:https://www.redhat.com/mailman/listinfo/open-scap-list[mailing list].
 
-Once you have forked the repository and decided what branch the fix will go into
-you can clone your forked repository.
+NOTE: The default branch of the openscap repository is set to the `maint-1.2`.
+
+First you need to clone your forked repository:
 [[app-listing]]
 [source,bash]
 ----
 $ git clone git@github.com:adam/openscap.git
 ----
 
-
-Now you can create a new branch, which we will call _fix_455_, where you can work on the fix. Remember that
-the name of the new branch will appear in the commit message when your fix is
-merged so choose the name wisely.
+Once you have forked the repository and decided that the fix will go into the
+`maint-1.2` branch you can create a new branch from it, which we will call
+_fix_455_, where you can work on the fix. Remember that the name of the new
+branch will appear in the commit message when your fix is merged so choose the
+name wisely:
 
 [[app-listing]]
 [source,bash]
 ----
-# checkout to the maint-1.0 branch
-$ git checkout maint-1.0
+$ cd openscap/
 # create a new branch
 $ git checkout -b fix_455
 ----
 
-NOTE: The default branch of the openscap repository is set to the `maint-1.2`.
-Our fix will go into the `maint-1.0` so we have to switch to this branch before
-creating a new one.
+NOTE: In the code snippets, lines starting with `#` are comments and lines
+starting with `$` are commands.
+
+On the other hand, if you decided that the fix will go into the `master` branch
+you have to switch to this branch first before creating a new one:
+[[app-listing]]
+[source,bash]
+----
+# create a new local branch to track the remote master branch
+$ git checkout -b master remotes/origin/master
```

Comment:
you will already have a local master tracking the origin master. this command is superfluous at best

Reply:
Are you sure about that? I documented this because when I do the clean clone of my forked repo I don't have it:
```
$ git clone https://github.com/matusmarhefka/openscap
$ cd openscap && git branch -a
* maint-1.2
  remotes/origin/HEAD -> origin/maint-1.2
  remotes/origin/maint-1.2
  remotes/origin/master
  remotes/origin/tests_doc
```